Sources/SwiftGodotMacroLibrary/MacroGodot.swift

```diff
@@ -37,7 +37,23 @@ class GodotMacroProcessor {
         
         // TODO: perhaps for these prop infos that are parameters to functions, we should not bother making them unique
         // and instead share all the Ints, all the Floats and so on.
-        ctor.append ("\tlet \(name) = PropInfo (propertyType: \(propType), propertyName: \"\(parameterName)\(parameterTypeName)\", className: className, hint: .none, hintStr: \"\", usage: .default)\n")
+        ctor.append ("\tlet \(name) = PropInfo (propertyType: \(propType), propertyName: \"\(parameterName)\", className: StringName(\"\(parameterTypeName)\"), hint: .none, hintStr: \"\", usage: .default)\n")
+        propertyDeclarations [key] = name
+        return name
+    }
+
+    func lookupPropReturn (parameterTypeName: String, parameterName: String) -> String {
+        let key = "\(parameterTypeName)/\(parameterName)"
+        if let v = propertyDeclarations [key] {
+            return v
+        }
+        let propType = godotTypeToProp (typeName: parameterTypeName)
```

Comment:
if `parameterTypeName` is `"Float"` then `propType` will be `".object"`, this is obvious not correct. Need to resolve this.

Reply:
similarly, if return type is `Float` for a swift func, the return type is an object of type `Float` in godot, it should just be `float`